pkg/distributor/distributor.go

```diff
@@ -28,6 +28,7 @@ import (
 	ingester_client "github.com/weaveworks/cortex/pkg/ingester/client"
 	"github.com/weaveworks/cortex/pkg/ring"
 	"github.com/weaveworks/cortex/pkg/util"
+	"google.golang.org/grpc/health/grpc_health_v1"
```

Comment:
Can you move this import up to the block above please?  The pattern ~is~ should be [std lib import, 3rd party imports, repo local import].  I thought this was standard (as described here https://github.com/golang/go/wiki/CodeReviewComments#imports) but its turns out it might just be me being picky.